doc/optimization/seeding.rst

```diff
@@ -0,0 +1,6 @@
+Seeding
+=======
+
+.. autoclass:: rtctools.optimization.seed_mixin.SeedMixin
```

Comment:
Refer to MultiSeedMixin.

It would also be good to discuss here that standard, "single" seeding functionality is available without requiring this mixin.

Reply:
Good catch! I'll update and double check the documentation